packages/superflare-remix/index.ts

```diff
@@ -51,8 +51,7 @@ export async function handleFetch<Env extends { APP_KEY: string }>(
     },
     async () => {
       /**
-       * We inject env and session into the Remix load context.
-       * Someday, we could replace this with AsyncLocalStorage.
+       * TODO: REMOVE THIS since we're using AsyncLocalStorage
```

Comment:
Or just deprecate it?

Reply:
For me, I think I'll still put some separate things in the context for now. But a lot of people probably never should need to care about context.